src/components/startup-modal/startup-modal.tsx

```diff
@@ -0,0 +1,39 @@
+import { Overlay } from "@blueprintjs/core";
+import { Trans, useTranslation } from "react-i18next";
+import { Links } from "../../config";
+import { useLocalStorage } from "../../hooks/use-local-storage";
+import { Modal } from "../modal";
+
+const MODAL_CLOSED_KEY = "vega_modal_closed";
+
+export const StartupModal = () => {
+  const { t } = useTranslation();
+  const [isOpen, setIsOpen] = useLocalStorage(MODAL_CLOSED_KEY, true);
+
+  return (
+    <Overlay
+      isOpen={isOpen}
```

Comment:
We may need a flag to enable/disable this, as if we deploy this to prod right now with staking turned off then people are going to be forced to read that bug message even though they cannot stake.

Reply:
Good catch. Will update